app/db_models.py

```diff
@@ -345,6 +347,64 @@ def update_obj(self, question, data):
             if field in data:
                 setattr(question, field, data[field])
 
+class FillInTheBlankQuestion(Question):
+    id = db.Column(db.Integer, db.ForeignKey('question.id'), primary_key=True)
+
+    question_text = db.Column(db.String, nullable=False)
+
+    answer = db.Column(db.String, nullable=False)
```

Comment:
The question may have multiple answers. If we are going to use a single column for all of the answers, there needs to be a way to separate each individual answer part in the string.